com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/LayersUIBlock.cs

```diff
@@ -17,15 +17,17 @@ public class LayersUIBlock : MaterialUIBlock
     {
         internal class Styles
         {
-            public static readonly string[] headers =
+            public static Vector2 layerIconSize = new Vector2(5, 5);
+
+            public static GUIContent[] layers { get; } =
             {
-                "Main layer",
-                "Layer 1",
-                "Layer 2",
-                "Layer 3",
+                EditorGUIUtility.TrTextContent(" Main layer", icon: Texture2D.whiteTexture),
+                EditorGUIUtility.TrTextContent(" Layer 1", icon: CoreEditorStyles.redTexture),
```

Comment:
There is a Texture2D.redTexture in Unity already so you don't need to create a new one:
https://docs.unity3d.com/ScriptReference/Texture2D-redTexture.html

Reply:
`Texture2D.redTexture` has alpha 0.

```
inline void Red(Texture2D *tex, unsigned char *data, int x, int y, int maxX, int maxY)
{
    data[0] = 255;
    data[1] = 0;
    data[2] = 0;
    data[3] = 0;
}
```

I need to create my own red without alpha.